streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/worker/tasks/StreamTask.java

```diff
@@ -5,98 +5,83 @@
 import java.util.List;
 import java.util.Map;
 import org.ray.api.Ray;
-import org.ray.api.RayActor;
 import org.ray.api.id.ActorId;
 import org.ray.streaming.api.collector.Collector;
 import org.ray.streaming.api.context.RuntimeContext;
-import org.ray.streaming.api.partition.Partition;
 import org.ray.streaming.runtime.core.collector.OutputCollector;
-import org.ray.streaming.runtime.core.graph.ExecutionEdge;
-import org.ray.streaming.runtime.core.graph.ExecutionGraph;
-import org.ray.streaming.runtime.core.graph.ExecutionNode;
+import org.ray.streaming.runtime.core.graph.executiongraph.ExecutionEdge;
+import org.ray.streaming.runtime.core.graph.executiongraph.ExecutionVertex;
 import org.ray.streaming.runtime.core.processor.Processor;
 import org.ray.streaming.runtime.transfer.ChannelID;
 import org.ray.streaming.runtime.transfer.DataReader;
 import org.ray.streaming.runtime.transfer.DataWriter;
 import org.ray.streaming.runtime.worker.JobWorker;
-import org.ray.streaming.runtime.worker.context.RayRuntimeContext;
-import org.ray.streaming.util.Config;
+import org.ray.streaming.runtime.worker.context.StreamingRuntimeContext;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+/**
+ * Task execution abstract class.
+ */
 public abstract class StreamTask implements Runnable {
```

Comment:
Can we add more documents for this class and all its methods?

Reply:
Thanks. Added more detailed comments.